### PR TITLE
framework/st_things: fixes indentation of definitions.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Aligns the value of definitions using space(' ') instead of tab('\t').

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1f67ca6cf1983bfb4d6fb1791600db7291c9a93d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1f67ca6cf1983bfb4d6fb1791600db7291c9a93d] - Code Rule Check OK.

**Comment by JeongJunSik** [comment]
in commit message, has wrong word.
please change istead to instead

File: framework/src/st_things/things_stack/cloud/cloud_connector.h [hunk 1]
```diff
@@ -93,5 +93,5 @@ extern OCStackResult things_cloud_topic_publish_topic(const char *host,
 		OCRepPayload *payload,
 		OCClientResponseHandler response);
 
-#endif	/* 
+#endif	/*
```
Comment (JeongJunSik) [inline framework/src/st_things/things_stack/cloud/cloud_connector.h]: I think it's better to erase this comment. (/* */)

Comment (Hyunjun85) [reply framework/src/st_things/things_stack/cloud/cloud_connector.h]: Thanks, I have fiexd.

**Comment by seinfra** [comment]
Target : [9a2fe1384543bbc5f065764067f9dfb069208178] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9a2fe1384543bbc5f065764067f9dfb069208178] - Code Rule Check OK.

File: framework/src/st_things/things_stack/cloud/cloud_manager.h [hunk 2]
```diff
@@ -25,19 +25,19 @@
 #include "framework/things_server_builder.h"
 #include <stdint.h>
 
-#define CI_COMMON_REFRESH_TIME      12	// access token Refresh period time without accesstoken-expire. (unit: hour)
+#define CI_COMMON_REFRESH_TIME      12		// access token Refresh period time without accesstoken-expire. (unit: hour)
 #define CI_TOKEN_EXPIRECHECK_TIME   3000	// access token Check period time whether it's expired or available. (unit: second)
```
Comment (JeongJunSik) [inline framework/src/st_things/things_stack/cloud/cloud_manager.h]: Align the line with the top line of comment

**Comment by JeongJunSik** [comment]
please align the below define also.
`#define` CLOUD_ERROR_CODE            "code"`

And it seems good to apply the following function parameters as well.

**Comment by seinfra** [comment]
Target : [dc8ce1ec625a144b8b8936ffabfddc45e5c1882f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [dc8ce1ec625a144b8b8936ffabfddc45e5c1882f] -

**Comment by seinfra** [comment]
Target : [5bec26077f96f068e553b2184a9719164c53ff42] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [5bec26077f96f068e553b2184a9719164c53ff42] - Code Rule Check (C++) OK.

**Pull request merged**